db/migrate/20220325174235_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- [ ] Please use a `text` instead of a `string` for the post's text because it may be longer

Reply:
Thank you for the suggestion. I will work on it.